lib/elixir/lib/calendar.ex

```diff
@@ -1439,7 +1439,7 @@ defmodule DateTime do
   Only supports converting date times which are in the ISO calendar,
   attempting to convert date times from other calendars will raise.
 
-  WARNING: the ISO 8601 does not contain the time zone nor its abbreviation,
+  WARNING: the ISO 8601 date time does not contain the time zone nor its abbreviation,
```

Comment:
Probably "format" reeds better.
I think we should use same wording in the function's summary as well.